src/routes/RequestsRoute.js

```diff
@@ -864,6 +953,19 @@ class RequestsRoute extends React.Component {
     Object.assign(requestData, { requestDate: isoDate });
   };
 
+  renderPaneSub() {
```

Comment:
Could you please clarify if we really need this functionality (this logic is absent in story description)?

Reply:
![image](https://github.com/folio-org/ui-requests/assets/127274721/3530e00e-7326-4889-b317-c017f65811b8)

Reply:
@rgupta-nla number of selected records is not described in the story/feature. Do we really need to implement it?